packages/IMAPClient-Core.package/ICCommand.class/class/fetchHeadersWithUIDFrom.to..st

```diff
@@ -0,0 +1,7 @@
+commands
+fetchHeadersWithUIDFrom: aBeginId to: anEndId
```

Comment:
Seems to be a misleading name, maybe use `fetchHeadersForUIDsFrom: aBeginUID to: anEndUID`